content/en/guides/models/track/log/distributed-training.md

```diff
@@ -100,6 +100,51 @@ Explore the W&B App UI to view an [example dashboard](https://wandb.ai/ayush-tha
 
 The preceding image demonstrates the W&B App UI dashboard. On the sidebar we see two experiments. One labeled 'null' and a second (bound by a yellow box) called 'DPP'. If you expand the group (select the Group dropdown) you will see the W&B Runs that are associated to that experiment.
 
+### Organize distributed runs with `job_type`
+
+When running distributed training with multiple processes, you can use `job_type` to distinguish between different types of nodes and create cleaner workspaces. This approach helps you organize your runs and filter out noise from worker nodes.
```

Comment:
```suggestion
When running distributed training with multiple processes, you can specify the `job_type` in your `wandb_init()` invocation, to distinguish between different types of nodes and create cleaner workspaces. This approach helps you organize your runs and filter out noise from worker nodes.
```
I checked and the job_type field is documented in the reference for init(). https://docs.wandb.ai/ref/python/sdk/functions/init/ WDYT about this edit?